modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -550,6 +550,13 @@ protected boolean isFieldWithConfirmationFieldAndVisible(
 			ddmFormEvaluatorFieldContextKey, "visible", true);
 	}
 
+	protected boolean isHideField(
+		DDMFormEvaluatorFieldContextKey ddmFormEvaluatorFieldContextKey) {
+
+		return getBooleanPropertyValue(
+			ddmFormEvaluatorFieldContextKey, "hideField", true);
```

Comment:
@brunofariasdeo I think you should set the default value to be `false` instead of `true`. Otherwise, if another field does not have this property it will return true. 
```
protected boolean getBooleanPropertyValue(
	DDMFormEvaluatorFieldContextKey ddmFormFieldContextKey,
	String booleanPropertyName, boolean defaultValue) 
```